hm_pyhelper/hardware_definitions.py

```diff
@@ -11,9 +11,17 @@ def is_raspberry_pi() -> bool:
     return sbc.is_sbc_type(sbc.DeviceVendorID.RASPBERRY_PI)
 
 
+# The variant name is determined by following rules:
+# - Format : <vendor>-<model>
+# - All lower case.
+# - Models are named v1,v2...vn, based on release chronology.
```

Comment:
This doesn't seem to be a good schema as it's mostly dictated by changes out of our control and calling it v1...v2 etc just chronologically on our side is confusing and conflicts with what some manufacturers call their own hardware

Reply:
@shawaj Sorry, I missed these comments as we were discussing other changes in a separate ticket.  This is what AC about openfleets mention. What alternative are we looking at? Should we do the actual model number inside the name, that will make too many fleets. 
We can try a name with <vendor>-<sbc> and then we can add another field inside the variant map which signifies actual models this variant and consequently this fleet supports.
eg.
SUPPORTED_MODELS = [ RAK-V1.5, RAK-V2, MNTD  ]

Will that work?

Reply:
@marvinmarnold @MuratUrsavas Any ideas on this.

Reply:
Well, my original idea on that was the same. We should name the variants based on products, not chronological order. You might remember this from the first meeting. If a product has a different name but based on the same variant, we could use the existing one without creating a new variant. If the any of the variant parameters has to change, then we should create a new one and name it after the product.

We can add SUPPORTED_MODELS list for preventing any upcoming confusions. But this time, it would mean unnecessary version bumps for every new addition to a list. I don't have a strong opinion about that.

Reply:
SUPPORTED_MODELS key has been added as part of https://github.com/NebraLtd/hm-pyhelper/pull/173 as I was already adding models and device keys for openfleets. That pull request builds on top of this.